frames/section.lua

```diff
@@ -222,6 +234,31 @@ local function onTitleClickOrDrop(section)
   events:SendMessage('bags/FullRefreshAll')
 end
 
+---@param section Section
+local function onTitleRightClick(section) 
```

Comment:
What happens if you spam right click?

Reply:
From what I just tested. What happens is... nothing. No error at all, the first click does what it needs to do and that's it.